lib/__tests__/harvest.test.ts

```diff
@@ -1,5 +1,5 @@
 import * as testServer from "../../server/server.js";
-import { getInstance } from "../../tests/utils";
+import { getInstance } from "../_instance";
```

Comment:
why  is this moved again here?

Reply:
`browser` and `node` modules were exporting an instance.
And we need "a function" that returns an instance in test cases.
So I had to have the functions in `_instance` file, so that I can use them in test cases
and also in `browser` and `node` module.